pkgs/top-level/python-packages.nix

```diff
@@ -11692,6 +11698,7 @@ self: super: with self; {
 
   tensorflowWithCuda = self.tensorflow.override {
     cudaSupport = true;
+    cudaPackages = pkgs.cudaPackages_11_7;
```

Comment:
https://github.com/NixOS/nixpkgs/blob/791b63b8ab8135e25d41e8d37b732b529bdd5b32/pkgs/top-level/python-packages.nix#L11678

Reply:
Or, I guess, https://github.com/NixOS/nixpkgs/blob/791b63b8ab8135e25d41e8d37b732b529bdd5b32/pkgs/top-level/python-packages.nix#L11672 then

Reply:
@nviets just in case I didn't make myself clear: I meant that we should pass `cudaPackages = cudaPackages_11_8` earlier in `tensorflow-bin = ...` already. Then `tensorflow` and `tensorflowWithCuda` will automatically pick this up, since `tensorflow` inherits `cudaPackages` from `tensorflow-bin`, and `tensorflowWithCuda` is just `tensorflow.override { ... }`

Can't say I endorse all of this indirection, but in the current state of things this is the way to make `(import nixpkgs { config.cudaSupport = true; }).tensorflow` work correctly (and to ensure our CI does build the cuda version)

Reply:
@nviets maybe you could operate this change so that I can directly test it on my hardware.

Reply:
Okay, done. Sorry for the long delay. I'm finally getting back to work. nixpkgs-review is on the way.